federation/pkg/federatedtypes/hpa.go

```diff
@@ -443,6 +447,10 @@ func (a *HpaAdapter) UpdateFederatedStatus(obj pkgruntime.Object, schedulingInfo
 			return fmt.Errorf("Error updating hpa: %s status in federation: %v", fedHpa.Name, err)
 		}
 	}
+
+	if err := a.updateClusterListOnTargetObject(fedHpa, schedulingInfo.(*hpaSchedulingInfo).scheduleState); err != nil {
+		return err
```

Comment:
I omitted that because, I was already formatting an error string in the functions `getRuntimeObjectForKind()` and `updateRuntimeObjectForKind()`. On second thoughts, and your suggestion, have augmented the string here also.